packages/config/config/devices/0x031e/lzw45.json

```diff
@@ -0,0 +1,301 @@
+		},
+		"21": {
+			"label": "Quick Strip Effect",
+			"description": "See website for details\n      *  Sending Basic, Binary Switch, Multilevel, and Color SET will automatically disable the strip effect.\n      *  Toggling OFF -> ON will cancel effect mode set on LED Strip and change the LED Strip back to the Previous color.\n      Range: 0 - 2147483647\n      Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 2147483647,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"22": {
+			"label": "Custom Strip Effect Parameter 1",
+			"description": "See website for details.\n      Range: 0 - 2147483647\n      Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 2147483647,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"23": {
+			"label": "Custom Strip Effect Parameter 2",
+			"description": "See website for details.\n      Range: 0 - 2147483647\n      Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 2147483647,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"24": {
+			"label": "Custom Strip Effect Parameter 3",
+			"description": "See website for details.\n      Range: 0 - 2147483647\n      Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 2147483647,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"30": {
+			"label": "Custom Strip Effect Parameter 4",
+			"description": "See website for details.\n      Range: 0 - 197375\n      Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 197375,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"31": {
+			"label": "Pixel Effect",
+			"description": "Byte 1 - Level\n      Byte 0 - Effect\n      See website for details.\n      Default: 0",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"51": {
+			"label": "Disable Physical On/Off Delay",
+			"description": "The 700ms delay that occurs after pressing the physical button to turn the switch on/off is removed. Consequently this also removes the following scenes: 2x, 3x, 4x, 5x tap. Still working are the 1x tap, held, released, and the level up/down scenes. (firmware 1.36+)\n      0 - 700ms delay disabled (some scenes will not work)\n      1 - 700ms delay enabled (All scenes work)\n      Default: 700ms delay enabled",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
```

Comment:
I think this could be a boolean

Reply:
Not possible currently, so this is fine.